### ISSUE TITLE
fiat values are way off

### ISSUE THREAD

**Issue by chappjc** [decred/dcrdex#1968]
just use mainnet for a while.

BTC typically ends up showing roughly half the USD after a while.

Showstopper for 0.6

**Comment by martonp** [comment]
That's just called a bear market.

**Comment by ukane-philemon** [comment]
> BTC typically ends up showing roughly half the USD after a while.

Can I see an image of how much it was?

**Comment by chappjc** [comment]
Usually about half, like $800 for 0.1 BTC.  I'll post if I see it again.  Has happened many times though.

**Comment by ukane-philemon** [comment]
> Usually about half, like $800 for 0.1 BTC. I'll post if I see it again. Has happened many times though.

Hmm, will look into it closely.

**Comment by ukane-philemon** [comment]
@chappjc, I've found the issue is with the rates from Messari. This makes me wonder if we should remove it as a source.

I ran dexc for the past 24 hours, printing rates to file for easy debugging and at some point, `Messari` decides to return 0 rates.

Coinparika 24 hour Rates: https://pastebin.com/Zakr7h52
Messarri 24 hour Rate: https://pastebin.com/cHgWAW6e
Dcrdata 24 hour Rates:  https://pastebin.com/X1hdNm18

Rates sent to the Frontend: https://pastebin.com/C8JjQJhr
